src/BenchmarkDotNet/Portability/Cpu/MosCpuInfoProvider.cs

```diff
@@ -33,7 +33,7 @@ private static CpuInfo Load()
                     processorsCount++;
                     physicalCoreCount += (uint) moProcessor[WmicCpuInfoKeyNames.NumberOfCores];
                     logicalCoreCount += (uint) moProcessor[WmicCpuInfoKeyNames.NumberOfLogicalProcessors];
-                    nominalClockSpeed = maxClockSpeed = minClockSpeed = (uint) moProcessor[WmicCpuInfoKeyNames.CurrentClockSpeed];
+                    nominalClockSpeed = maxClockSpeed = minClockSpeed = (uint) moProcessor[WmicCpuInfoKeyNames.MaxClockSpeed];
```

Comment:
I think it would be better to set only `maxClockSpeed`.
`nominalClockSpeed` can be parsed from the CPU brand string.
It also makes sense to don't set `minClockSpeed` on Windows (because we don't know how to get the right value).